UPGRADE-3.4.md

```diff
@@ -114,6 +114,35 @@ FrameworkBundle
    class has been deprecated and will be removed in 4.0. Use the
    `Symfony\Component\Translation\DependencyInjection\TranslatorPass` class instead.
 
+HttpKernel
+----------
+
+ * Relying on commands auto-discovery has been deprecated and won't be supported
+   in 4.0. Use PSR-4 based Service Discovery instead.
+
+   Before:
+
+   ```yml
+   # app/config/services.yml
+   services:
+       # ...
+
+       # implicit commands registration
+   ```
+
+   After:
+
+   ```yml
+   # app/config/services.yml
+   services:
+       # ...
+
+       # explicit commands registration
+       AppBundle\Command:
+           resource: '../../src/AppBundle/Command/*'
+           tags: ['console.command']
```

Comment:
Not required when using auto configuration. Given the [default config](https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/config/services.yaml#L16-L22) that'll be promoted, should we really add this before/after at all? Or at least mention it's not required when using the new default config?